helm/ako/templates/configmap.yaml

```diff
@@ -13,6 +13,7 @@ data:
   cloudName: {{ .Values.ControllerSettings.cloudName | quote }}
   clusterName: {{ .Values.AKOSettings.clusterName | quote }}
   servicesAPI: {{ .Values.AKOSettings.servicesAPI | quote }}
+  enableEVH: {{ .Values.AKOSettings.enableEVH | quote }}
```

Comment:
Ideally I dont want the users to be setting this flag ever, unless they want to migrate from older model to EVH model, or if we hit any bugs.
Can we have a logic to figure out what should be the value of this field when not set by the user.

Reply:
Yes. I am still working on the logic, will add with the rest layer code. Currently I have disabled it by default as only graph  layer changes are there in the PR.